backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -221,21 +221,17 @@ public AccessRequest GetAccessRequest(int id)
         /// <param name="quantity"></param>
         /// <param name="sort"></param>
         /// <param name="status"></param>
-        public Paged<AccessRequest> GetAccessRequests(int page = 1, int quantity = 10, string sort = null, AccessRequestStatus? status = null)
+        public Paged<AccessRequest> GetAccessRequests(int page = 1, int quantity = 10, string sort = null,
+            AccessRequestStatus status = AccessRequestStatus.OnHold)
         {
-            this.User.ThrowIfNotAuthorized(Permissions.AdminUsers);
-
-            var query = this.Context.AccessRequests
-                .Include(p => p.Agencies)
-                .ThenInclude(p => p.Agency)
-                .Include(p => p.Roles)
-                .ThenInclude(p => p.Role)
-                .Include(p => p.User)
-                .AsNoTracking();
+            var sortArray = new string[] { };
+            if (!string.IsNullOrWhiteSpace(sort))
+            {
+                sortArray[0] = sort;
+            }
 
-            query = query.Where(request => request.Status == status);
-            var accessRequests = query.Skip((page - 1) * quantity).Take(quantity);
-            return new Paged<AccessRequest>(accessRequests, page, quantity, query.Count());
+            var filter = new AccessRequestFilter(page, quantity, sortArray, null, null, null, status);
```

Comment:
Could the latter 4 parameters default to null? that would let you clean this up a bit.

Reply:
You could also use the implementation in other filters where it'll parse the query string instead of manually passing arguments in a specific order.